processor/tasks/CROWNMultiFriends.py

```diff
@@ -17,47 +17,24 @@
 
 
 class CROWNMultiFriends(CROWNExecuteBase):
-    output_collection_cls = law.NestedSiblingFileCollection
```

Comment:
Why is this not needed anymore? I remember that this was needed for multiFriend production because otherwise there were problems with correctly tracking the jobs if more than one scope was used. I'm not an expert, so if this now works, I'm fine.

Reply:
All Tasks iherit from the base Task in framework.py, where the same argument is included:
https://github.com/KIT-CMS/KingMaker/blob/ba00dbc0d46d459767bcc0e67cea0acc6b6a3988/processor/framework.py#L69

I removed this because it defines the same thing twice.

Reply:
Not right now. I just remember that we had some problems and this was part of the solution. But I didn't saw that it is already defined elsewhere, so it's should be fine.